docs/rewards.md

```diff
@@ -0,0 +1,36 @@
+---
+id: rewards
+title: Rewards
```

Comment:
I would prefer to call it `Incentives` but perhaps it is better to leave it as rewards

Reply:
Maybe staking rewards is better since we'll have more different rewards